### PR TITLE
fix: Fix binding for APIs during git import

### LINKED ISSUE #12328: [Bug]: The API binding fails when user import app from Git
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

The API Import fails when the user import it from git
image

### Steps To Reproduce

1. Try to import the application which is connected to git 
2. Make sure the application has an API
3. Now inside the imported app The Api binding fails and user cant rerun the same API

### Public Sample App

_No response_

### Version

release

### PR DESCRIPTION
**Pull request by AnaghHegde**
## Description

> The actions of type API binding fails when the user import it from git. This is introduced recently due to the fix raised for the visibility of git connected apps. It is because of the missing org id. Affects git import, git checkout remote branch and git pull. 

Fixes #12328 

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Locally
- JUnit tets

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/server/appsmith-server/src/test/resources/test_assets/ImportExportServiceTest/valid-application-with-page-added.json [hunk 1]
```diff
@@ -176,16 +543,1192 @@
   ],
   "publishedDefaultPageName": "Page1",
   "unpublishedDefaultPageName": "Page1",
-  "actionList": [],
+  "actionList": [
```
Comment (abhvsn) [inline app/server/appsmith-server/src/test/resources/test_assets/ImportExportServiceTest/valid-application-with-page-added.json:546]: Can we please have a dummy data if possible for these static file?

**Review by abhvsn — APPROVED** [review]
LGTM

**Comment by AnaghHegde** [comment]
@abhvsn Can you please approve the PR again ?

**Review by abhvsn — APPROVED** [review]

**Pull request merged**